Automattic/wp-calypso
# Plans 2023: Use usePlans hook in pricing meta for grid plans

Related to https://github.com/Automattic/wp-calypso/issues/79005

## Proposed Changes

- Replace `usePricedAPIPlans` with `usePlans` hook from plans data store. Removes the `use-priced-api-plans` implementations. 
- Updates `billingPeriod` and `currencyCode` in pricing-meta to be derived from it.

## Testing Instructions



* Ensure pricing in `/plans/[ site ]` and `/start/plans` renders correctly. Anything around bill period (billing descriptions) and currency codes.

## Pre-merge Checklist
